src/main/resources/templates/modal/syncDestModal.html

```diff
@@ -7,7 +7,8 @@ <h5 class="modal-title" id="sync-dest-modal">Synchronization Destination Confirm
 </div>
 <div class="modal-body" id="modal-body">
     <div class="alert alert-warning" role="alert" th:text="#{screen.message.modal.preferences.warning}"></div>
-    <p th:utext="#{screen.message.common.syncDest.confirmationModalText}"></p>
+    <div>{{ confirmationMessage }}</div>
+    <b>{{ syncDestDescription }}</b>
```

Comment:
Could you rename the $scope variable confirmationMessage to something like syncDestConfirmationMessage to match the naming of $scope.syncDestDescription. Thanks!

Reply:
You may have forgotten to update `{{ confirmationMessage }}` to `{{ syncDestConfirmationMessage}}`.